oap-server/server-starter/src/main/resources/ui-initialized-templates/faas/faas-function.json

```diff
@@ -323,12 +323,6 @@
                   "metricTypes": [
                     "sortMetrics"
                   ],
-                  "metricConfig": [
-                    {
-                      "sortOrder": "DES",
```

Comment:
Should we keep DES? As this is a metric of traffic? More traffic more important?

Reply:
`DES` is the default order